app/lib/dns.server.ts

```diff
@@ -113,11 +113,23 @@ export const upsertRecord = async (
 ) => {
   try {
     if (!isNameValid(name, username)) {
-      throw new Error('Invalid name provided');
+      logger.error('Invalid record name provided', {
+        name,
+        username,
+        baseDomian: `${username}.${process.env.ROOT_DOMAIN}`,
```

Comment:
There is a small typo in spelling here; I wonder if it will be a problem now or later.

Reply:
Nice catch!

No, it's just logging, I'll fix it in the next PR